### PR TITLE
Send the data received to on_ipc_error event.

### PR DESCRIPTION
**Pull request by MeFaltaUnVerano21**
### Description
Send the data received to on_ipc_error event.

- [x] This PR has been tested.

### CODE REVIEW

File: discord/ext/ipc/server.py [hunk 1]
```diff
@@ -189,7 +189,7 @@ async def handle_accept(self, request: aiohttp.web.Request):
                             endpoint,
                             request,
                         )
-                        self.bot.dispatch("ipc_error", endpoint, error)
+                        self.bot.dispatch("ipc_error", request.get("data", None), endpoint, error)
```
Comment (Gobot1234) [inline discord/ext/ipc/server.py:192]: dict.get by default returns None so the second argument is pointless

**Review by MeFaltaUnVerano21 — COMMENTED** [review]
Sry, fixed

**Comment by Gobot1234** [comment]
This is also a breaking change

**Review by ShineyDev — CHANGES_REQUESTED** [review]
Hey, thanks for the PR! As [mentioned above](https://github.com/Ext-Creators/discord-ext-ipc/pull/25#issuecomment-792274902) this isn't going to be merged as a breaking change, but that isn't to say that you cannot remedy it.

I suggest you discuss the design with @lgaan when he's around, but I have a few questions myself:
- What made you want to pass data through to the handler?
- Why should the data come before the endpoint in the handler call? Generally speaking, getting more specific with data as you go is correct.
- Why is data alone so important? I think passing the `request` object itself would be more useful.

**Comment by lgaan** [comment]
I'll wait for the above questions to be answered. But for now my stance is that this is not needed.

**Comment by lgaan** [comment]
I feel like a better design, and non breaking fix, for this would be to attach a `request` arg to the error parsed to the event.

**Comment by lgaan** [comment]
Opening a pull request to implement this with the design above later. With that I will close this issue now.

**Pull request closed without merging**